jfrog-cli/artifactory/commands/config.go

```diff
@@ -124,19 +127,39 @@ func getConfigurationFromUser(details, defaultDetails *config.ArtifactoryDetails
 		ioutils.ScanFromConsole("Artifactory URL", &details.Url, defaultDetails.Url)
 		allowUsingSavedPassword = false
 	}
+	// Ssh-Key
 	if fileutils.IsSshUrl(details.Url) {
-		if err := getSshKeyPath(details); err != nil {
+		return getSshKeyPath(details)
+	}
+	// Api-Key/Password/Access-Token
+	if details.ApiKey == "" && details.Password == "" && details.AccessToken == "" {
+		err := readAccessTokenFromConsole(details)
+		if err != nil {
 			return err
 		}
-
-	} else {
-		if details.ApiKey == "" && details.Password == "" {
-			ioutils.ReadCredentialsFromConsole(details, defaultDetails, allowUsingSavedPassword)
+		if len(details.GetAccessToken()) == 0 {
+			return ioutils.ReadCredentialsFromConsole(details, defaultDetails, allowUsingSavedPassword)
 		}
 	}
 	return nil
 }
 
+func readAccessTokenFromConsole(details *config.ArtifactoryDetails) error {
+	print("Access token (Leave blank for username and password): ")
```

Comment:
I suggest changing to:
Access token (Leave blank for username and password/**API key**):